robocop/checkers/lengths.py

```diff
@@ -71,6 +78,15 @@ def visit_File(self, node):
     def visit_Keyword(self, node):  # noqa
         if node.name.lstrip().startswith('#'):
             return
+        for child in node.body:
+            if isinstance(child, Arguments):
```

Comment:
I will verify it

Reply:
Ok, now I had time to go back to this and I haven't mentioned that this rule only checks the definitions of the keywords, not keyword calls. Do you think we should check also keyword calls?

Reply:
You're right, I thought it was also checking keyword calls hence my question. I think definition is enough, otherwise it would complain on every usage of our 'too many arguments' keywords.